microsoft/MixedRealityToolkit-Unity
# Fixing Null Reference Exception in SystemKeyboardExample

## Overview

When running the HandInteractionExamples the Unity log is spammed with null reference exceptions from the SystemKeyboardExample due to the debug text not being set up in the inspector. This pull request address this by assigning the reference and performing runtime checks.

## Changes
- Fixes: https://github.com/microsoft/MixedRealityToolkit-Unity/issues/5501


## Verification

1. Run the HandInteractionExamples on a HoloLens 2.
2. Download the UnityPlayer.log
3. Verify no exceptions from Microsoft.MixedReality.Toolkit.Examples.Demos.SystemKeyboardExample.Update

>
> As a reviewer, it is possible to check out this change locally by using the following
> commands (substituting {PR_ID} with the ID of this pull request):
>
> git fetch origin pull/{PR_ID}/head:name_of_local_branch
>
> git checkout name_of_local_branch
